### PR TITLE
[calceph] update to 3.5.2

### PR DESCRIPTION
**Pull request by gastineau**
- [ X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ X] SHA512s are updated for each updated download
- [ X] The "supports" clause reflects platforms that may be fixed by this new version
- [ X] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ X] Any patches that are no longer applied are deleted from the port's directory.
- [ X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [ X] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/calceph/portfile.cmake [hunk 1]
```diff
@@ -1,5 +1,5 @@
-set(CALCEPH_VERSION "3.5.1")
-set(CALCEPH_HASH 5e83bb46b92a0b53f2cae717363cb4497d5c9cb57b3903e70d9e2c50176ca7d234212d0209fd3fcb5feebfd0980313be17e2ad4e69482504bfe8686f93216b67)
+set(CALCEPH_VERSION "3.5.2")
```
Comment (LilyWangLL) [inline ports/calceph/portfile.cmake:1]:
```suggestion
```

File: ports/calceph/portfile.cmake [hunk 2]
```diff
@@ -1,5 +1,5 @@
-set(CALCEPH_VERSION "3.5.1")
-set(CALCEPH_HASH 5e83bb46b92a0b53f2cae717363cb4497d5c9cb57b3903e70d9e2c50176ca7d234212d0209fd3fcb5feebfd0980313be17e2ad4e69482504bfe8686f93216b67)
+set(CALCEPH_VERSION "3.5.2")
+set(CALCEPH_HASH 4e0b702494c9fc05f6a192671bccdc0f8002b5706cba4bd7ff50d9e8754203b4a78e3759ccc3003809123584c9bca1434411e111357852dbc58e4d1903c1879e)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-${CALCEPH_VERSION}.tar.gz"
```
Comment (LilyWangLL) [inline ports/calceph/portfile.cmake:5]:
```suggestion
    URLS "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-${VERSION}.tar.gz"
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/calceph/portfile.cmake (hunk 1 above)
Comment (gastineau) [reply ports/calceph/portfile.cmake:1]: CALCEPH_VERSION is required in the next part of the script

File: ports/calceph/portfile.cmake (hunk 2 above)
Comment (gastineau) [reply ports/calceph/portfile.cmake:5]: VERSION is never defined .

Comment (gastineau) [reply ports/calceph/portfile.cmake:4]: done

File: ports/calceph/portfile.cmake [hunk 3]
```diff
@@ -1,9 +1,8 @@
-set(CALCEPH_VERSION "3.5.1")
-set(CALCEPH_HASH 5e83bb46b92a0b53f2cae717363cb4497d5c9cb57b3903e70d9e2c50176ca7d234212d0209fd3fcb5feebfd0980313be17e2ad4e69482504bfe8686f93216b67)
+set(CALCEPH_HASH 4e0b702494c9fc05f6a192671bccdc0f8002b5706cba4bd7ff50d9e8754203b4a78e3759ccc3003809123584c9bca1434411e111357852dbc58e4d1903c1879e)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-${CALCEPH_VERSION}.tar.gz"
```
Comment (LilyWangLL) [inline ports/calceph/portfile.cmake:4]:
```suggestion
    URLS "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-${VERSION}.tar.gz"
```
Please change the version in download URL.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Review by LilyWangLL — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**